com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceShadows.cs

```diff
@@ -183,7 +185,7 @@ public override void OnCameraCleanup(CommandBuffer cmd)
                     throw new ArgumentNullException("cmd");
                 }
 
-                cmd.ReleaseTemporaryRT(m_RenderTarget.id);
+                cmd.ReleaseTemporaryRT(Shader.PropertyToID(m_RenderTarget.name));
```

Comment:
Either caching the id or replacing the use of Get/Release TemporaryRT

Reply:
I'm not sure if there will be some conflict with `ScreenSpaceShadowResolvePass.cs` which uses the same name for the target but with different filter mode.

Reply:
Then let's cache it :D 

> conflict with `ScreenSpaceShadowResolvePass.cs`

I'm not following entirely. Can you elaborate on what the conflict is?

Reply:
These [[1]](https://github.com/Unity-Technologies/Graphics/blob/883bd1c/com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceShadows.cs#L108-L114) [[2]](https://github.com/Unity-Technologies/Graphics/blob/883bd1c/com.unity.render-pipelines.universal/Runtime/Passes/ScreenSpaceShadowResolvePass.cs#L20) two passes initialize render textures with the same name. `Shader.PropertyToID` will result in the same ids and since they both do `GetTemporaryRT`, they're conflicting.

When converting them to RTHandles, I run into this conflict: Do I create one RTHandle which is shared by these two or do I create two RTHandles with the same name?

[1] https://github.com/Unity-Technologies/Graphics/blob/883bd1c/com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceShadows.cs#L108-L114

[2] https://github.com/Unity-Technologies/Graphics/blob/883bd1c/com.unity.render-pipelines.universal/Runtime/Passes/ScreenSpaceShadowResolvePass.cs#L20

Reply:
I think one RTHandle that is shared would be best since they refer to the same texture